credentialdigger/client.py

```diff
@@ -1144,14 +1144,18 @@ def _scan(self, repo_url, scanner, models=None, force=False, debug=False,
         discoveries_ids = list()
         if debug:
             logger.debug('Update database with these discoveries.')
-            for i in tqdm(range(len(new_discoveries))):
-                curr_d = new_discoveries[i]
-                new_id = self.add_discovery(
-                    curr_d['file_name'], curr_d['commit_id'],
-                    curr_d['line_number'], curr_d['snippet'], repo_url,
-                    curr_d['rule_id'], curr_d['state'])
-                if new_id != -1 and curr_d['state'] != 'false_positive':
-                    discoveries_ids.append(new_id)
+            with Progress() as progress:
+                inserting_task = progress.add_task('Inserting discoveries...',
+                                                   total=len(new_discoveries))
+                for curr_discovery in new_discoveries:
+                    curr_d = curr_discovery
```

Comment:
not needed... use the same variable! :D

Reply:
if you don't want to update the references below, call it `curr_d` in the for loop...

Reply:
You are totally right, sorry what a noob mistake :)